{
    "Computer":  "VSPS1",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1760700460486)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  173116
                   },
                   {
                       "TimeCreated":  "\/Date(1760700388330)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Access to performance data was denied to user \"SYSTEM\" (value from GetUserName() for the running thread) as attempted from module \"C:\\Program Files\\SentinelOne\\Sentinel Agent 24.2.3.471\\SentinelAgent.exe\" (value from GetModuleFileName() for the binary that issued the query).",
                       "Id":  1000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  30991
                   },
                   {
                       "TimeCreated":  "\/Date(1760700380417)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The Sentinel Agent service did not shut down properly after receiving a preshutdown control.",
                       "Id":  7043,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  218094
                   },
                   {
                       "TimeCreated":  "\/Date(1760700168194)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to launch action \"%windir%\\system32\\devicecensus.exe\" in instance \"{cddd1157-065b-4e05-8b2b-4a817589deb5}\" of task \"\\Microsoft\\Windows\\Device Information\\Device\". Additional Data: Error Value: 2147943515.",
                       "Id":  203,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  173097
                   },
                   {
                       "TimeCreated":  "\/Date(1760698849924)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{2f085fb2-8649-4d89-8529-ff31c5497e8a}\" of \"\\Microsoft\\Windows\\Wininet\\CacheTask\"  task for user \"\" . Additional Data: Error Value: 2147942404.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  173092
                   },
                   {
                       "TimeCreated":  "\/Date(1760698849924)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\Wininet\\CacheTask\" , instance \"{2f085fb2-8649-4d89-8529-ff31c5497e8a}\" , action \"Wininet Cache task object\" . Additional Data: Error Value: 2147942404.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  173091
                   },
                   {
                       "TimeCreated":  "\/Date(1760628173523)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The SysMain service terminated with the following error: \r",
                       "Id":  7023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  216284
                   }
               ],
    "LastPoll":  "10/17 04:42"
}